app.py

```diff
@@ -21,6 +22,19 @@
 # init MySQL
 mysql = MySQL(app)
 
+#init Mail
+#Ensure 2 Factor Authentication is off and allow less secure app access
+#Enter sender email id in routes sending mail also
+mail = Mail(app)
+
+app.config['MAIL_SERVER'] = 'smtp.gmail.com'
+app.config['MAIL_PORT'] = 465
+app.config['MAIL_USERNAME'] = '********@gmail.com'
+app.config['MAIL_PASSWORD'] = '***********'    
+app.config['MAIL_USE_TLS'] = False
+app.config['MAIL_USE_SSL'] = True
+mail = Mail(app)
+
```

Comment:
To be added in `config.py`.
Can be merged as it is for now since the file is missing in this branch.